helm-charts/helm3/strimzi-kafka-operator/templates/050-ConfigMap-strimzi-cluster-operator-logging.yaml

```
@@ -0,0 +1,28 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: {{ .Values.logConfigMap }}
+data:
+  log4j2.properties: |
+    name = COConfig
```

```suggestion
    name = ClusterOperatorConfig
```

brevity here seems unnecessary